CONTRIBUTING.md

```diff
@@ -193,6 +193,12 @@ easy to reference the pull request where a change was introduced.
 - [squash](https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git) merge pull request
 - the `unstable` branch may be used to aggregate pull merges before fixing tests
 
+### Git Commit Style
+
+Write concise commits that finish the sentence "This change modifies Tendermint to...". Most of
+the time, one PR should have one commit. More information on good commit messages can be found
```

Comment:
```suggestion
one PR should have one commit once it lands on `master`; this can be accomplished by using the "squash and merge" button on Github. Be sure to edit your commit message, though!